src/jimgw/core/single_event/likelihood.py

```
@@ -968,7 +970,772 @@ def _step(carry, _):
+    """
+
+    highest_mode: int
+    accuracy_factor: Float
+    reference_chirp_mass: Float
+    reference_chirp_mass_in_second: Float
+    time_offset: Float
+    delta_f_end: Float
+    max_banding_frequency: Float
+    min_banding_duration: Float
+
+    durations: Float[Array, " n_bands"]
```

Ruff (0.15.13)

[error] Syntax error in forward annotation: Expected an expression

(F722)